### PR TITLE
feat(starfish): add legend to linegraph

### PR DESCRIPTION
**Pull request by DominikB2014**
If there is more there 2 series, a legend will be added to the linegraph. useful for throughput chart in database module
image

### CODE REVIEW

File: static/app/views/starfish/modules/databaseModule/databaseChartView.tsx [hunk 1]
```diff
@@ -75,19 +75,21 @@ export default function APIModuleView({}: Props) {
   const data = Object.values(seriesByAction);
 
   const seriesByQuery: {[action: string]: Series} = {};
-  topGraphData.forEach(datum => {
-    seriesByQuery[datum.description] = {
-      seriesName: datum.description.substring(0, 50),
-      data: [],
-    };
-  });
+  if (!isTopGraphLoading) {
```
Comment (DominikB2014) [inline static/app/views/starfish/modules/databaseModule/databaseChartView.tsx:78]: Also added this conditonal, which kept giving me occasional runtime errors when trying process data that's not loading.

File: static/app/views/starfish/components/chart.tsx [hunk 2]
```diff
@@ -284,6 +286,14 @@ function Chart({
...
               colors={colors}
               grid={grid}
+              {...(showLineGraphChart
+                ? {
+                    legend: {
+                      top: 0,
+                      right: 10,
+                    },
+                  }
+                : {})}
```
Comment (0Calories) [inline static/app/views/starfish/components/chart.tsx:296]: nit: This is cool lol but a bit unnecessary, I think we can just do something like:
And did you mean to call your variable something like `showLegend`?
```suggestion
              legend={showLineGraphChart ? {top: 0, right: 10} : undefined}
```

**Review by 0Calories — APPROVED** [review]
Nice! Just have a nit, feel free to ignore

File: static/app/views/starfish/components/chart.tsx (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/starfish/components/chart.tsx]: Yeah good point @0Calories, I swear I tried that and it didn't work, but now it does lol.

Also, changed the variable name to `showLineGraphLegend`.

**Pull request merged**